scripts/entrypoint.pl

```diff
@@ -156,6 +156,7 @@ sub cmd_test_webservices {
 
 sub cmd_test_selenium {
     my $conf = require $ENV{BZ_QA_CONF_FILE};
+    $ENV{HTTP_BACKEND} = 'simple';
```

Comment:
perhaps it would be better to run tests under the same backend as prod?

Reply:
The selenium tests don't run with any level of concurrency and haven't for a long time.
For apache the number of workers as also limited. I'm confident hypnotoad and the simple httpd are close enough for this.